src/breadcrumb/Breadcrumb.unit.tsx

```diff
@@ -20,9 +20,17 @@ describe('Breadcrumb', () => {
     expect(screen.getAllByRole('listitem')).toHaveLength(1)
     expect(screen.getByRole('list')).toHaveTextContent('Fhtagn')
   })
-  it('renders with 2 links and a separator', () => {
+  it('renders with 2 links and a separator between them', () => {
     render(<Breadcrumb crumbs={[crumb1, crumb2]} />)
     expect(screen.getAllByRole('listitem')).toHaveLength(2)
     expect(screen.getByRole('list')).toHaveTextContent('Fhtagn › Cthulhu')
   })
+  it('renders with no divider ', () => {
+    render(<Breadcrumb crumbs={[crumb1]} />)
+    expect(screen.queryByRole('separator')).not.toBeInTheDocument()
+  })
+  it('renders with a divider on top', () => {
+    render(<Breadcrumb crumbs={[crumb1]} divider={DividerPosition.TOP} />)
+    expect(screen.queryByRole('separator')).toBeInTheDocument()
```

Comment:
you should check the order of the elements on the DOM to make sure its before or after

Reply:
Done